snapshots/fasttrack/latest/table_5__threatened_species_in_each_major_group_by_country__show_all__1.csv.dvc

```diff
@@ -1,10 +1,9 @@
 meta:
   namespace: fasttrack
-  short_name: table_5__threatened_species_in_each_major_group_by_country__show_all__1
```

Comment:
Even though it looks ugly, changing `short_name` would create a new dataset (which is problematic because the existing one has charts). So either you keep it as it is or you'll have to match old variables to the new ones.